Parse/src/main/java/com/parse/Parse.java

```diff
@@ -643,37 +629,24 @@ private static ParseEventuallyQueue getEventuallyQueue(Context context) {
     }
   }
 
-  static void checkInit() {
```

Comment:
Ah, I saw that it was unused, and since it was not public, I figured it was safe to delete. I did not realize that ParseLiveQuery was of the same package name and would be accessing things that were package protected within Parse. I'll add it back, but it would probably be best to annotate or comment on things where this is the case